common/src/main/resources/schema/development/example.json

```diff
@@ -0,0 +1 @@
+{"example": "json"}
```

Comment:
When https://github.com/DataBiosphere/terra-data-catalog/pull/157 is merged, this can be replaced with a real example dataset.